httpd.c

```diff
@@ -128,6 +129,25 @@ char *request_header(const char *name) {
 // get all request headers
 header_t *request_headers(void) { return reqhdr; }
 
+// Handle escape characters (%xx)
+static void uri_unescape(char *s)
+{
+  char c = 0;
+  char *t = s;
```

Comment:
`t` means `result`?

Reply:
t for destination (changed in dst, which should be clearer).